web/themes/new_weather_theme/templates/partials/observation-table.html.twig

```diff
@@ -5,7 +5,7 @@
             <th scope="row"  class="font-family-mono text-base font-mono-xs text-uppercase z-400">
                 {{ "Wind" | t }}
             </th>
-            <td>
+            <td class="padding-left-0">
```

Comment:
Woops, I think you applied `padding-left` instead of `padding-right` in this commit